compete/rc-spring.html

```diff
@@ -123,62 +123,103 @@
+            <!-- Put Game Day description in JS above -->
+            <div class="kickoff-info"></div>
+            <div class="kickoff-description"></div>
+          </div>
 
-        <a class="list-group-item" data-toggle="collapse" data-target="#dr-panel-description" data-parent="#timeline-panel">Design Reviews<span class="glyphicon glyphicon-plus pull-right"></span></a>
-        <div id="dr-panel-description" class="sublinks collapse"><!-- Put DR description in JS above --><div class="dr-info">
-        </div><div class="dr-description">
-        </div></div>
+          <div class="tab-pane fade in" id="game-day">
+            <h1 class="blue-title">Game Day</h1>
+            <!-- Put Game Day description in JS above -->
+            <div class="game-day-info"></div>
+            <div class="game-day-description"></div>
+          </div>
 
-        <a class="list-group-item" data-toggle="collapse" data-target="#scrimmage-panel-description" data-parent="#timeline-panel">Scrimmage<span class="glyphicon glyphicon-plus pull-right"></span></a>
-        <div id="scrimmage-panel-description" class="sublinks collapse"><!-- Put Scrimmage description in JS above --><div class="scrimmage-info">
-        </div><div class="scrimmage-description">
-        </div></div>
+          <div class="tab-pane fade in" id="dr">
+            <h1 class="blue-title">Design Reviews</h1>
+            <!-- Put Game Day description in JS above -->
+            <div class="dr-info"></div>
+            <div class="dr-description"></div>
+          </div>
 
-        <a class="list-group-item" data-toggle="collapse" data-target="#final-comp-panel-description" data-parent="#timeline-panel">Final Competition<span class="glyphicon glyphicon-plus pull-right"></span></a>
-        <div id="final-comp-panel-description" class="sublinks collapse"><!-- Put Final Competition description in JS above --><div class="finalcomp-info">
-        </div><div class="final-comp-description">
-        </div></div>
+          <div class="tab-pane fade in" id="scrimmage">
+            <h1 class="blue-title">Scrimmage</h1>
+            <!-- Put Game Day description in JS above -->
+            <div class="scrimmage-info"></div>
+            <div class="scrimmage-description"></div>
+          </div>
+
+          <div class="tab-pane fade in" id="final-comp">
+            <h1 class="blue-title">Final Competition</h1>
+            <!-- Put Game Day description in JS above -->
+            <div class="final-comp-info"></div>
+            <div class="final-comp-description"></div>
+          </div>
+        </div>
       </div>
     </div>
   </div>
-  <hr class="horizontal-border">
-  <!-- Schedule -->
-  <div class="row text-center">
-    <h2 class="blue-title"> Season Schedule </h2>
+
+  <!-- gcal link -->
+  <div class="row" style="text-align: right;">
+     <a href="../compete/gcal.html">Add Google Calendar <span class="glyphicon glyphicon-calendar"></span></a>
```

Comment:
Is it possible to have this say something more "appealing" to a user? Or even just more clear, like "Google Calendar of Season Events"? Maybe I'm just a derp but I had originally read this as if we were missing a google calendar and one needed to be added.